### PR TITLE
MULTIARCH-4000 - Create CI job for ABI on power

### PR DESCRIPTION
**Pull request by cs-zhang**

### CODE REVIEW

**Comment by cs-zhang** [comment]
/pj-rehearse  periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-agent-sno-power

**Comment by cs-zhang** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-agent-sno-power

**Comment by cs-zhang** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-agent-sno-power

**Comment by cs-zhang** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-agent-sno-power

**Comment by cs-zhang** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-agent-sno-power

**Comment by cs-zhang** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-agent-sno-power

**Comment by cs-zhang** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-agent-sno-power

**Comment by cs-zhang** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-agent-sno-power

**Comment by cs-zhang** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-agent-sno-power

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml [hunk 1]
```diff
@@ -732,6 +732,14 @@ tests:
       INSTALL_TYPE: sno
       OCP_VERSION: "4.15"
     workflow: upi-powervs-sno
+- as: ocp-e2e-agent-sno-power
+  cron: 0 1 * * 1
```
Comment (tvardema) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml:736]: Can we confirm the cron timings so we don't run into lease starvation?  PowerVS leases are limited, from what I understand, so you can potentially talk with Shilpa/MikeT to get the right timing here.

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.16.yaml [hunk 2]
```diff
@@ -710,6 +710,14 @@ tests:
       INSTALL_TYPE: sno
       OCP_VERSION: "4.16"
     workflow: upi-powervs-sno
+- as: ocp-e2e-agent-sno-power
+  cron: 0 1 * * 1
```
Comment (tvardema) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.16.yaml:714]: Same as other cron comment.

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml (hunk 1 above)
Comment (cs-zhang) [reply ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml:736]: OK. let me confirm it with them.

Comment (cs-zhang) [reply ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml:736]: After talked with Shilpa/MikeT, there is no problem with the timing for ABI SNO.

**Comment by cs-zhang** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-agent-sno-power

**Comment by cs-zhang** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-agent-sno-power

**Comment by cs-zhang** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-agent-sno-power

**Pull request merged**